packages/client/src/object/convertWireToOsdkObjects.ts

```diff
@@ -153,6 +155,38 @@ export function convertWireToOsdkObjects<
   objs: OntologyObjectV2[],
 ) {
   for (const obj of objs) {
+    if (obj.__rid) {
+      obj.$rid = obj.__rid;
+      delete obj.__rid;
+    }
+
+    // Backend returns as __apiName but we want to stick to $ structure
+    obj.$apiName = obj.__apiName;
+
+    // for now these are the same but when we start doing interface projections the $objectType will always be underlying and
+    // the $apiName will be for the current view (in current designs)
+    obj.$objectType = obj.__apiName;
+
+    // copying over for now as its always returned. In the future, this should just be inferred from underlying
+    obj.$primaryKey = obj.__primaryKey;
+
+    // After Feb 2024 (unless we have new apis):
```

Comment:
So what happens after this month, do we just set that const to true and we delete those fields?

Reply:
Yeah, just trying to give some time for people to deal with breaks.